### PR TITLE
Added verbose mode

### PR DESCRIPTION
**Pull request by Breush**

### CODE REVIEW

**Comment by Breush** [comment]
Incoming: `--quiet` option

File: yoga/cli.py [hunk 1]
```diff
@@ -15,6 +15,18 @@ def add_main_cli_arguments(parser):
             help="Output file path",
             type=argparse.FileType("wb")
             )
+    parser.add_argument(
+            "--verbose",
```
Comment (flozz) [inline yoga/cli.py]: please add a short option too (`-v`)

File: yoga/cli.py [hunk 2]
```diff
@@ -15,6 +15,18 @@ def add_main_cli_arguments(parser):
...
+            help="enable verbose mode",
+            default=False,
+            action="store_true"
+            )
+    parser.add_argument(
+            "--quiet",
```
Comment (flozz) [inline yoga/cli.py]: please add a short option too (`-q`)

File: yoga/model/__init__.py [hunk 3]
```diff
@@ -1,11 +1,13 @@
 from .assimp import (assimp_import_from_bytes, assimp_export_to_bytes)
 from .options import (normalize_options, extract_image_options)
+from ..options import normalize_options as normalize_global_options
 from .helpers import model_embed_images
 
 import os.path
 
 
 def optimize(input_file, output_file, options={}):
```
Comment (flozz) [inline yoga/model/__init__.py]: quiet ad verbose are not realy options of the optimize function as it does not change the output, maybe can be better to add them this way:

```python
def optimize(input_file, output_file, options={}, quiet=False, verbose=False):
```

**Pull request merged**